src/env.ts

```diff
@@ -22,6 +22,8 @@ const schema = Type.Object(
     TURBO_TOKEN: Type.String({ separator: ',' }),
     PORT: Type.Number({ default: 3000 }),
     LOG_LEVEL: Type.Optional(Type.String({ default: 'info' })),
+    LOG_TYPE: Type.Optional(Type.String({ default: 'stdout' })),
+    LOG_DIR: Type.Optional(Type.String({ default: 'server.log' })),
```

Comment:
calling it `LOG_DIR` even if it is a file?

Reply:
if you override this variable you have to specify the full filename, even with the extension. otherwise it creates you a server.log file in the homedir. maybe `LOG_FILE` as suggested on issue?

Reply:
Changed to:

LOG_MOD and LOG_FILE